projects/example-app/src/app/auth/reducers/auth.reducer.spec.ts

```diff
@@ -21,10 +23,11 @@ describe('AuthReducer', () => {
     });
   });
 
+  /*
```

Comment:
Why is this test commented out?

Reply:
The loginPageActions.Login action is not used in the reducer under test.

Reply:
Ok, just remove it then